docs/README.md

```diff
@@ -20,6 +20,7 @@
     * [Platform Specific Tag Confines](dsl/platform_specific_tag_confines.md)
     * [Shared Options for Executing Beaker Commands](Shared-Options-for-Executing-Beaker-Commands.md)
   * [Let's Write a Test!](Lets-Write-a-Test.md)
+  
```

Comment:
just waiting for this unnecessary line to be taken out at this point. Otherwise, it looks good.